api/auth.py

```diff
@@ -327,10 +356,82 @@ def post(self):
                 'message': 'Error, user does not exist.',
             }
             return make_response(jsonify(responseObject)), 401
+
+
+class RoomReserved(MethodView):
+    """
+    User Resource
+    """
+    from service.reserve_room_service import get_reservation, get_meditatoin_room_by_id, get_room_reserved_by_id, get_reservation_by_id, add_room_reserved_return_id, add_reservation_return_id
+
+    def post(self):
+        # get auth token
+        auth_header = request.headers.get('Authorization')
+        if auth_header:
+            auth_token = auth_header.split(" ")[1]
+        else:
+            auth_token = ''
+
+        if auth_token:
+            resp = Reservation.decode_auth_token(auth_token)
+            if isinstance(resp, str):
+                post_data = request.get_json()
+
+                meditation_room_id = post_data.get('meditation_room_id')
+                employee_id = post_data.get('employee_id')
```

Comment:
We shouldn't be passing employee id in the request body. Rather it should be identified from auth token